src/bepasty/utils/_compat.py

```diff
@@ -11,7 +11,7 @@
 if PY2:
     from urlparse import urlparse, urljoin
 
-    range_type = xrange
+    range_type = xrange  # noqa: F821
```

Comment:
did you run the pep8 checker on py2 and py3? this is somehow asymetric now.

Reply:
Yes I did. Of course it's asymetric, `range` is perfectly fine keyword in Py2. The other way - not.

Reply:
ah, right.